Node server for client app

chawlarajan opened the pull request:
Gupreet, please merge this with your code.

chawlarajan committed:
data source configuration

chawlarajan committed:
Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction

chawlarajan committed:
Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction

chawlarajan committed:
removed unwanted packages, install ngx-own-carousel

chawlarajan committed:
created node server to serve client app, added socket module to listen order notifications

Gurpeet commented:
Merged.

On Fri, Jul 14, 2017 at 5:47 AM, chawlarajan <notifications@github.com>
wrote:

> Gupreet, please merge this with your code.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/Gurpeet/PizzaJunction/pull/3
> Commit Summary
>
>    - data source configuration
>    - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>    - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>    - removed unwanted packages, install ngx-own-carousel
>    - created node server to serve client app, added socket module to
>    listen order notifications
>
> File Changes
>
>    - *M* api/server/datasources.json
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-0> (6)
>    - *A* client/package-lock.json
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-1> (4399)
>    - *M* client/package.json
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-2> (9)
>    - *M* client/src/app/shared/components/globals/global.ts
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-3> (3)
>    - *A* server/package-lock.json
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-4> (602)
>    - *A* server/package.json
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-5> (16)
>    - *A* server/server.js
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-6> (38)
>    - *A* server/socket.js
>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-7> (20)
>
> Patch Links:
>
>    - https://github.com/Gurpeet/PizzaJunction/pull/3.patch
>    - https://github.com/Gurpeet/PizzaJunction/pull/3.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Gurpeet/PizzaJunction/pull/3>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFxxhA2ndpKTaCVk10GPAYMO1BSmT9xXks5sNrOrgaJpZM4OXsH9>
> .
>



-- 
Regards,
Gurpreet Singh

Gurpeet commented:
Implemented bootstrap carousel. Removed ng2-own-carousel and
ngx-own-carousel.
Pl let me know if they were being used.

Thanks,
Gurpreet Singh


On Fri, Jul 14, 2017 at 8:34 AM, Gurpreet Singh <gopisivia@gmail.com> wrote:

> Merged.
>
> On Fri, Jul 14, 2017 at 5:47 AM, chawlarajan <notifications@github.com>
> wrote:
>
>> Gupreet, please merge this with your code.
>> ------------------------------
>> You can view, comment on, or merge this pull request online at:
>>
>>   https://github.com/Gurpeet/PizzaJunction/pull/3
>> Commit Summary
>>
>>    - data source configuration
>>    - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>>    - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>>    - removed unwanted packages, install ngx-own-carousel
>>    - created node server to serve client app, added socket module to
>>    listen order notifications
>>
>> File Changes
>>
>>    - *M* api/server/datasources.json
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-0> (6)
>>    - *A* client/package-lock.json
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-1> (4399)
>>    - *M* client/package.json
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-2> (9)
>>    - *M* client/src/app/shared/components/globals/global.ts
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-3> (3)
>>    - *A* server/package-lock.json
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-4> (602)
>>    - *A* server/package.json
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-5> (16)
>>    - *A* server/server.js
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-6> (38)
>>    - *A* server/socket.js
>>    <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-7> (20)
>>
>> Patch Links:
>>
>>    - https://github.com/Gurpeet/PizzaJunction/pull/3.patch
>>    - https://github.com/Gurpeet/PizzaJunction/pull/3.diff
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Gurpeet/PizzaJunction/pull/3>, or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AFxxhA2ndpKTaCVk10GPAYMO1BSmT9xXks5sNrOrgaJpZM4OXsH9>
>> .
>>
>
>
>
> --
> Regards,
> Gurpreet Singh
>



-- 
Regards,
Gurpreet Singh

chawlarajan commented:
Nope, bootstrap carousel should be good enough.

On Thu, Jul 13, 2017 at 10:10 PM Gurpeet <notifications@github.com> wrote:

> Implemented bootstrap carousel. Removed ng2-own-carousel and
> ngx-own-carousel.
> Pl let me know if they were being used.
>
> Thanks,
> Gurpreet Singh
>
>
> On Fri, Jul 14, 2017 at 8:34 AM, Gurpreet Singh <gopisivia@gmail.com>
> wrote:
>
> > Merged.
> >
> > On Fri, Jul 14, 2017 at 5:47 AM, chawlarajan <notifications@github.com>
> > wrote:
> >
> >> Gupreet, please merge this with your code.
> >> ------------------------------
> >> You can view, comment on, or merge this pull request online at:
> >>
> >> https://github.com/Gurpeet/PizzaJunction/pull/3
> >> Commit Summary
> >>
> >> - data source configuration
> >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
> >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
> >> - removed unwanted packages, install ngx-own-carousel
> >> - created node server to serve client app, added socket module to
> >> listen order notifications
> >>
> >> File Changes
> >>
> >> - *M* api/server/datasources.json
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-0> (6)
> >> - *A* client/package-lock.json
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-1> (4399)
> >> - *M* client/package.json
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-2> (9)
> >> - *M* client/src/app/shared/components/globals/global.ts
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-3> (3)
> >> - *A* server/package-lock.json
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-4> (602)
> >> - *A* server/package.json
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-5> (16)
> >> - *A* server/server.js
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-6> (38)
> >> - *A* server/socket.js
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-7> (20)
> >>
> >> Patch Links:
> >>
> >> - https://github.com/Gurpeet/PizzaJunction/pull/3.patch
> >> - https://github.com/Gurpeet/PizzaJunction/pull/3.diff
> >>
> >> —
> >> You are receiving this because you are subscribed to this thread.
> >> Reply to this email directly, view it on GitHub
> >> <https://github.com/Gurpeet/PizzaJunction/pull/3>, or mute the thread
> >> <
> https://github.com/notifications/unsubscribe-auth/AFxxhA2ndpKTaCVk10GPAYMO1BSmT9xXks5sNrOrgaJpZM4OXsH9
> >
> >> .
> >>
> >
> >
> >
> > --
> > Regards,
> > Gurpreet Singh
> >
>
>
>
> --
> Regards,
> Gurpreet Singh
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Gurpeet/PizzaJunction/pull/3#issuecomment-315271388>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcbBWBRj2isu45G56VunMZrKr1CTiTC9ks5sNvgrgaJpZM4OXsH9>
> .
>
-- 
Rajan-

Gurpeet commented:
Hi Rajan,

Lets have chat when you available.
Due to some personal engagements I was not able to work for a couple of
days.
I started yesterday and installed/updated the dependencies and found
ng-pick-datetime was breaking the address page. After some investigation
found that it has been updated and some major changes have been made.
Please refer Update section in "
https://www.npmjs.com/package/ng-pick-datetime"
I have made the required changes and now delivery time control is visible
but still it is not binding to the control and so selected value is not
showing up.

Also I have added interceptor to the application. It will help us to
modify/add information to all http requests. We will use it when some
header will be required to appended to the requests and for logging. But
had to do considerable changes for this. Please refer to my last commit.
Now moving back to:
1) Generating POST request for add ContactUs.
2) Creating table and sproc for ContactUs.
3) Calculating distance on Go Next and display delivery charges as per
distance.

Queries:
1) Do we need to save Delivery Address at the time of click of Go Next or
when checking out.
2) Should Delivery time control be at Delivery address page as currently
the distance has not be calculated and it might be possible person selects
time to be 45 mins from now but the delivery time is more due to distance.

Thanks,
Gurpreet Singh

On Fri, Jul 14, 2017 at 10:41 AM, chawlarajan <notifications@github.com>
wrote:

> Nope, bootstrap carousel should be good enough.
>
> On Thu, Jul 13, 2017 at 10:10 PM Gurpeet <notifications@github.com> wrote:
>
> > Implemented bootstrap carousel. Removed ng2-own-carousel and
> > ngx-own-carousel.
> > Pl let me know if they were being used.
> >
> > Thanks,
> > Gurpreet Singh
> >
> >
> > On Fri, Jul 14, 2017 at 8:34 AM, Gurpreet Singh <gopisivia@gmail.com>
> > wrote:
> >
> > > Merged.
> > >
> > > On Fri, Jul 14, 2017 at 5:47 AM, chawlarajan <notifications@github.com
> >
> > > wrote:
> > >
> > >> Gupreet, please merge this with your code.
> > >> ------------------------------
> > >> You can view, comment on, or merge this pull request online at:
> > >>
> > >> https://github.com/Gurpeet/PizzaJunction/pull/3
> > >> Commit Summary
> > >>
> > >> - data source configuration
> > >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
> > >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
> > >> - removed unwanted packages, install ngx-own-carousel
> > >> - created node server to serve client app, added socket module to
> > >> listen order notifications
> > >>
> > >> File Changes
> > >>
> > >> - *M* api/server/datasources.json
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-0> (6)
> > >> - *A* client/package-lock.json
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-1> (4399)
> > >> - *M* client/package.json
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-2> (9)
> > >> - *M* client/src/app/shared/components/globals/global.ts
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-3> (3)
> > >> - *A* server/package-lock.json
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-4> (602)
> > >> - *A* server/package.json
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-5> (16)
> > >> - *A* server/server.js
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-6> (38)
> > >> - *A* server/socket.js
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-7> (20)
> > >>
> > >> Patch Links:
> > >>
> > >> - https://github.com/Gurpeet/PizzaJunction/pull/3.patch
> > >> - https://github.com/Gurpeet/PizzaJunction/pull/3.diff
> > >>
> > >> —
> > >> You are receiving this because you are subscribed to this thread.
> > >> Reply to this email directly, view it on GitHub
> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3>, or mute the thread
> > >> <
> > https://github.com/notifications/unsubscribe-auth/
> AFxxhA2ndpKTaCVk10GPAYMO1BSmT9xXks5sNrOrgaJpZM4OXsH9
> > >
> > >> .
> > >>
> > >
> > >
> > >
> > > --
> > > Regards,
> > > Gurpreet Singh
> > >
> >
> >
> >
> > --
> > Regards,
> > Gurpreet Singh
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/Gurpeet/PizzaJunction/pull/3#issuecomment-315271388
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AcbBWBRj2isu45G56VunMZrKr1CTiTC9ks5sNvgrgaJpZM4OXsH9>
> > .
> >
> --
> Rajan-
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Gurpeet/PizzaJunction/pull/3#issuecomment-315271589>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFxxhBGr-UP9ZNFqE99q6S14XC0EyFU2ks5sNviPgaJpZM4OXsH9>
> .
>



-- 
Regards,
Gurpreet Singh

Gurpeet commented:
Hi Rajan,

Worked today with distance calculation and picked the code you provided and
implemented the same.
After completing implementation got the following message:

[image: Inline image 1]

Which more seems like configuration issue. Was just wondering if you are
aware of it.
Have committed the code for same.

Thanks,
Gurpreet Singh

On Tue, Jul 18, 2017 at 11:22 AM, Gurpreet Singh <gopisivia@gmail.com>
wrote:

> Hi Rajan,
>
> Lets have chat when you available.
> Due to some personal engagements I was not able to work for a couple of
> days.
> I started yesterday and installed/updated the dependencies and found
> ng-pick-datetime was breaking the address page. After some investigation
> found that it has been updated and some major changes have been made.
> Please refer Update section in "https://www.npmjs.com/
> package/ng-pick-datetime"
> I have made the required changes and now delivery time control is visible
> but still it is not binding to the control and so selected value is not
> showing up.
>
> Also I have added interceptor to the application. It will help us to
> modify/add information to all http requests. We will use it when some
> header will be required to appended to the requests and for logging. But
> had to do considerable changes for this. Please refer to my last commit.
> Now moving back to:
> 1) Generating POST request for add ContactUs.
> 2) Creating table and sproc for ContactUs.
> 3) Calculating distance on Go Next and display delivery charges as per
> distance.
>
> Queries:
> 1) Do we need to save Delivery Address at the time of click of Go Next or
> when checking out.
> 2) Should Delivery time control be at Delivery address page as currently
> the distance has not be calculated and it might be possible person selects
> time to be 45 mins from now but the delivery time is more due to distance.
>
> Thanks,
> Gurpreet Singh
>
> On Fri, Jul 14, 2017 at 10:41 AM, chawlarajan <notifications@github.com>
> wrote:
>
>> Nope, bootstrap carousel should be good enough.
>>
>> On Thu, Jul 13, 2017 at 10:10 PM Gurpeet <notifications@github.com>
>> wrote:
>>
>> > Implemented bootstrap carousel. Removed ng2-own-carousel and
>> > ngx-own-carousel.
>> > Pl let me know if they were being used.
>> >
>> > Thanks,
>> > Gurpreet Singh
>> >
>> >
>> > On Fri, Jul 14, 2017 at 8:34 AM, Gurpreet Singh <gopisivia@gmail.com>
>> > wrote:
>> >
>> > > Merged.
>> > >
>> > > On Fri, Jul 14, 2017 at 5:47 AM, chawlarajan <
>> notifications@github.com>
>> > > wrote:
>> > >
>> > >> Gupreet, please merge this with your code.
>> > >> ------------------------------
>> > >> You can view, comment on, or merge this pull request online at:
>> > >>
>> > >> https://github.com/Gurpeet/PizzaJunction/pull/3
>> > >> Commit Summary
>> > >>
>> > >> - data source configuration
>> > >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>> > >> - Merge branch 'master' of https://github.com/Gurpeet/PizzaJunction
>> > >> - removed unwanted packages, install ngx-own-carousel
>> > >> - created node server to serve client app, added socket module to
>> > >> listen order notifications
>> > >>
>> > >> File Changes
>> > >>
>> > >> - *M* api/server/datasources.json
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-0> (6)
>> > >> - *A* client/package-lock.json
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-1>
>> (4399)
>> > >> - *M* client/package.json
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-2> (9)
>> > >> - *M* client/src/app/shared/components/globals/global.ts
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-3> (3)
>> > >> - *A* server/package-lock.json
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-4> (602)
>> > >> - *A* server/package.json
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-5> (16)
>> > >> - *A* server/server.js
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-6> (38)
>> > >> - *A* server/socket.js
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3/files#diff-7> (20)
>> > >>
>> > >> Patch Links:
>> > >>
>> > >> - https://github.com/Gurpeet/PizzaJunction/pull/3.patch
>> > >> - https://github.com/Gurpeet/PizzaJunction/pull/3.diff
>> > >>
>> > >> —
>> > >> You are receiving this because you are subscribed to this thread.
>> > >> Reply to this email directly, view it on GitHub
>> > >> <https://github.com/Gurpeet/PizzaJunction/pull/3>, or mute the
>> thread
>> > >> <
>> > https://github.com/notifications/unsubscribe-auth/AFxxhA2ndp
>> KTaCVk10GPAYMO1BSmT9xXks5sNrOrgaJpZM4OXsH9
>> > >
>> > >> .
>> > >>
>> > >
>> > >
>> > >
>> > > --
>> > > Regards,
>> > > Gurpreet Singh
>> > >
>> >
>> >
>> >
>> > --
>> > Regards,
>> > Gurpreet Singh
>> >
>> > —
>> > You are receiving this because you authored the thread.
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/Gurpeet/PizzaJunction/pull/3#issuecomment-315271388
>> >,
>> > or mute the thread
>> > <https://github.com/notifications/unsubscribe-auth/AcbBWBRj2
>> isu45G56VunMZrKr1CTiTC9ks5sNvgrgaJpZM4OXsH9>
>> > .
>> >
>> --
>> Rajan-
>>
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Gurpeet/PizzaJunction/pull/3#issuecomment-315271589>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AFxxhBGr-UP9ZNFqE99q6S14XC0EyFU2ks5sNviPgaJpZM4OXsH9>
>> .
>>
>
>
>
> --
> Regards,
> Gurpreet Singh
>



-- 
Regards,
Gurpreet Singh
